modules/apigee-organization/variables.tf

```diff
@@ -43,6 +45,15 @@ variable "apigee_environments" {
     condition     = alltrue([for k, v in var.apigee_environments : contains(["DEPLOYMENT_TYPE_UNSPECIFIED", "PROXY", "ARCHIVE"], v.deployment_type)])
     error_message = "Allowed values for deployment_type \"DEPLOYMENT_TYPE_UNSPECIFIED\", \"PROXY\" or \"ARCHIVE\"."
   }
+
+  validation {
+    condition     = alltrue([for k, v in var.apigee_environments : v.min_node_count >= 2])
```

Comment:
can you split in multiple lines. See [here](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/modules/vpc-sc/variables.tf#L90) for example.